libs/wingsdk/src/http/http.ts

```diff
@@ -134,6 +135,11 @@ export interface RequestOptions {
    * @default about:client
    */
   readonly referrer?: string;
+  /**
+   * Timeout for terminating a pending request. None if undefined.
+   * @default undefined
```

Comment:
Can you check if Node's `fetch` has a default timeout? I think it might be 30s - but if there isn't any then let's add "no timeout"
```suggestion
   * @default - 30s
```
```suggestion
   * @default - no timeout
```

Reply:
there isn't any- otherwise I could just add the property :)